README.md

```diff
@@ -1,197 +1,22 @@
-# Kernel-Based Hough Transform for Detecting Straight Lines in Images
+# Kernel-Based Hough Transform for Detecting Straight Lines in Images for the LSST Stack
 
-This repository contains the reference implementation of the Kernel-Based Hough Transform (KHT). The KHT is a real-time line detection procedure that extends the conventional voting procedure of the [Hough transform](https://en.wikipedia.org/wiki/Hough_transform). It operates on clusters of approximately collinear pixels. For each cluster, the KHT casts votes using an oriented elliptical-Gaussian kernel that models the uncertainty associated with the best-fitting line for the corresponding cluster. The proposed approach not only significantly improves the performance of the voting scheme, but also produces a much cleaner voting map and makes the transform more robust to the detection of spurious lines.
+This is the LSST stack code fork of the [KHT package](https://github.com/laffernandes/kht). Please see [Fernandes and Oliveira, 2008](http://www.ic.uff.br/~laffernandes/content/publications/journal/2008_pr_41(1)/fernandes_oliveira-pr-41(1)-2008-pre_print.pdf), for a full description of the algorithm.
```

Comment:
It might be nice/polite to include the citation for their paper here in case anyone else uses this package.